pygbif/occurrences/download.py

```diff
@@ -1,10 +1,12 @@
 import os
 import csv
 import json
+import re
 import datetime
-from requests import auth
+import requests
+import pygbif
```

Comment:
`import pygbif` ? I've not seen this in a python package before, importing the package that you're already in? curious what does that do?

Reply:
Yeah, I don't like it much neither. I think a good explanation lies there: https://stackoverflow.com/questions/42862042/in-python-how-to-import-from-the-init-py-in-the-same-directory

I'll try to rearrange things a bit further so we don't need to import from init.py.